### ISSUE TITLE
Rogue Talents - Turn the Tables and Honor Among Thieves

### ISSUE THREAD

**Issue by alexuktc** [dalaranwow/dalaran-wow#5927]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
1. Turn the Tables - The talent gives the rogue 2/4/6% increased Critical Strike chance on his special skills whenever someone in his party or raid dodges or parries an attack. 
The issue I noticed is that if you have a rogue with this talent in your raid all rogues will receive the buff. I have not tested if it actually works and whether it works with druids in cat form as well.

2. Honor Among Thieves - old issue, several issues with it:
2.1. It will not add combo points to you if you do not already have a combo point on your target.
2.2. The skill has no internal cooldown. It can add all 4 combo points (since you have to have at least 1 CP for it to work as mentioned above) in less than a second.
2.3. It works for all energy users, meaning that all rogues and feral druids in cat form receive the same bonus. 
2.4. It works for the entire raid when it should only work for critical strikes from people in your group.

**Expected behaviour**: Tell us what should happen instead
1. The skill should only work for the rogue that has it.

2. The skill should work even if you don't have combo points on your target, it should have an internal cooldown, only work for you and should only proc from people in your party group.

**Steps to reproduce the problem**:

1. Test
2. ??
3. Profit

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
